pkg/subctl/cmd/gather.go

```diff
@@ -0,0 +1,124 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	_ "github.com/submariner-io/lighthouse/test/e2e/discovery"
+	_ "github.com/submariner-io/lighthouse/test/e2e/framework"
+	_ "github.com/submariner-io/submariner/test/e2e/dataplane"
+	_ "github.com/submariner-io/submariner/test/e2e/redundancy"
```

Comment:
why do we need those?

Reply:
We don't. I missed removing these from my scratch code.